src/cli/commands/migrate.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+import { type ChangelogAPIType } from '../../types';
+
+export default async function runMigrate(
```

Comment:
I thought about the fact that migration is – at its core – a pretty isolated, reusable logic
umzug seems way over the top (at least what's in the initial sections here seems to suggest that: https://www.npmjs.com/package/umzug#configuration)
In the end, the most code here is involved with reading/writing the `info.yml` and actually testing the migrations – do you feel a library would save us much trouble here?